app/javascript/components/widgets/twitter/tweet_body.jsx

```diff
@@ -10,6 +10,8 @@ import {
 } from '@twitter/tweet_content';
 import { colors, fonts, fontSizes } from '../../../lib/theme';
 
+const Entities = require('html-entities').XmlEntities;
```

Comment:
Prefer es6 import style

Reply:
It seems that import style doesn't seem to work. It completely breaks that way.
https://www.npmjs.com/package/html-entities

Reply:
@sracca I think there's just a little mismatch between the idioms from each way of importing. 

```javascript
const Entities = require('html-entities').AllHtmlEntities;
```

Is saying: **1** include the `'html-entities'` module, **2** get the `AllHtmlEntities` from it, and **3** _rename_ it to "`Entities`". 

In `import` terms, that'd look like:

**1** Import the module (we won't use it like this; just explaining the mapping):
```javascript
import * as HtmlEntities from 'html-entities';
```
**2** To get the `AllHtmlEntities` member, we'd do: 
```javascript
import { AllHtmlEntities } from 'html-entities'`
```
**3** Finally, to do the rename:
```javascript
import { AllHtmlEntities as Entities } from 'html-entities';
```

**3** _should_ work here